pkg/controller/performanceprofile/performanceprofile_controller.go

```diff
@@ -304,6 +313,7 @@ func (r *ReconcilePerformanceProfile) Reconcile(request reconcile.Request) (reco
 		return *result, nil
 	}
 
+	r.recorder.Eventf(instance, corev1.EventTypeNormal, "Creation succeeded", "Succeeded to create all components")
```

Comment:
won't this event get posted every time the reconcile loop pops after everything is stable?

it seems like a better option might be to post this in the applyComponents function only in the situation that the final component change was made. so not when all components already exist, but when we actually finish successfully posting all components. 

that way the event only triggers once, or only one per a mutation of the profile object.

Reply:
done